src/acsets/acsets.py

```diff
@@ -544,13 +544,25 @@ def import_pydantic(cls, name: str, schema: Schema, d: Any):
 
         return acs
 
-    def write_json(self):
+    def to_json_obj(self):
+        """Serialize the ACSet to a JSON object.
+
+        Returns:
+            The JSON object of the serialized ACSet.
+        """
+        return self.export_pydantic().dict()
+
+    def to_json_file(self, fname, *args, **kwargs):
+        with open(fname, 'w') as fh:
+            fh.write(self.write_json(*args, **kwargs))
+
+    def write_json(self, *args, **kwargs):
```

Comment:
this name doesn't make sense - if it's returning a string then it's not being written anywhere

Reply:
Yeah. It makes sense to swap the names to_json and write_json

Reply:
This naming predates my PR so I haven't changed it. Should I change it in this PR or do that separately?

Reply:
I wouldn't mind changing it in this PR.

Reply:
Okay, done, we now have 3 consistently named methods: `to_json_obj`, `to_json_str`, and `to_json_file`, each useful in various settings.